commander/src/utils/core/pm2.ts

```diff
@@ -258,10 +258,16 @@ export const listApplication = async (): Promise<
 
 export const describeApplication = async (
 	name: string,
-): Promise<PM2ProcessInstance> => {
-	await connectPM2();
-	const application = (await describePM2(name)) as PM2ProcessInstance;
-	disconnect();
-
-	return extractProcessDetails(application);
+): Promise<PM2ProcessInstance | boolean> => {
+	try {
+		await connectPM2();
+		const application = await describePM2(name);
+		disconnect();
+
+		return extractProcessDetails(application);
+	} catch (error) {
+		disconnect();
+
+		return false;
```

Comment:
prefer to return consistent type.
maybe
`Promise<PM2ProcessInstance | undefined>` or rethrow error is bit better

Reply:
the problem with throwing would be difficult to handle the case.. here i am returning boolean, that way I can handle the condition, something like this
```
const instance = await describeApplication(name);

		if (!instance) {
			this.log(
				`Lisk Core instance: ${name} doesn't exists, Please install using lisk core:install`,
			);

			return;
		}
		// tslint:disable-next-line await-promise
		await StopCommand.run([name]);
		// tslint:disable-next-line await-promise
		await StartCommand.run([name]);
```

Reply:
then let's say `undefined` =)